tokio/README.md

```diff
@@ -155,9 +155,28 @@ several other libraries, including:
 
 ## Supported Rust Versions
 
-Tokio is built against the latest stable release. The minimum supported version is 1.45.
-The current Tokio version is not guaranteed to build on Rust versions earlier than the
-minimum supported version.
+Tokio is built against the latest stable release. The minimum supported version
+is 1.45.  The current Tokio version is not guaranteed to build on Rust versions
+earlier than the minimum supported version.
+
+## Release schedule
+
+Tokio doesn't follow a fixed release schedule, but we typically make one to two
+new minor releases each month. We make patch releases for bugfixes as necessary.
+
+## Bug patching policy
+
+For the purposes of making patch releases with bugfixes, we have designated
+certain minor releases as LTS (long term support) releases. Whenever a bug
+warrants a patch release with a fix for the bug, it will be backported and
+released as a new patch release for each LTS minor version. Our current LTS
+releases are:
```

Comment:
Do we expect to have multiple? I think in general the convention is to have _one_ LTS release that expires when the next one rolls around?

Reply:
I was thinking we would have two for around a month of overlap when one expires.

Reply:
Do you think it'd be worthwhile calling that out explicitly here?

Reply:
Perhaps a better policy is every three months. Then we have two LTS releases at any one time (at least start three months from now).